### PR TITLE
Data fetched from master table inside patient and doctor components

### PR DESCRIPTION
**Pull request by niluroy**
1) Languages, locations, qualifications, and consultation modes data fetched from master tables. Required services added inside shared service to fetch data.

### CODE REVIEW

File: src/client/app/profile/doctor-profile.component.ts [hunk 1]
```diff
@@ -91,7 +87,44 @@ export class DoctorProfileComponent implements OnInit {
             allowSearchFilter: true
           };
     }
-      update({ value }: { value: any }) {
+
+    getLocations() {
+        this.sharedService.getLocations()
+            .subscribe(locations => {
+                locations.map((location: any) => {
```
Comment (arun-awnics) [inline src/client/app/profile/doctor-profile.component.ts]: Isn't location of type string here?

File: src/client/app/profile/patient-profile.component.ts [hunk 2]
```diff
@@ -77,13 +80,40 @@ export class PatientProfileComponent implements OnInit {
...
                     documentTitle: null,
                     documentDescription: null
                 });
             });
     }
 
+    getAllergies() {
+        this.sharedService.getAllergies()
+            .subscribe(allergies => {
+                allergies.map((allergy: any) => {
```
Comment (arun-awnics) [inline src/client/app/profile/patient-profile.component.ts]: Similarly in this component as well. Check the type

Comment (niluroy) [reply src/client/app/profile/patient-profile.component.ts]: allergy also is a JSON object so it cant be of string type.

File: src/client/app/profile/doctor-profile.component.ts (hunk 1 above)
Comment (arun-awnics) [reply src/client/app/profile/doctor-profile.component.ts]: Noted

File: src/client/app/profile/patient-profile.component.ts (hunk 2 above)
Comment (arun-awnics) [reply src/client/app/profile/patient-profile.component.ts]: Alright

**Pull request merged**